src/Adapter/MSTest.TestAdapter/SourceGeneratedDynamicDataOperations.cs

```diff
@@ -0,0 +1,8 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter;
+
+internal class SourceGeneratedDynamicDataOperations : DynamicDataOperations
```

Comment:
This will be needed later for data driven tests probably, having extra class allows me to have all the initializations wired correctly already.